libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/util/AuthzHelper.java

```diff
@@ -47,12 +48,19 @@ static ObjectMapper initJsonMapper() {
         return objectMapper;
     }
 
-    public static void removeRoleMembers(List<RoleMember> originalRoleMembers, List<RoleMember> removeRoleMembers) {
+    private static boolean isUpdateRequired(RoleMember member1, RoleMember member2) {
+        return !Objects.equals(member1.getExpiration(), member2.getExpiration()) ||
+                !Objects.equals(member1.getReviewReminder(), member2.getReviewReminder());
+    }
```

Comment:
Why not use the RoleMember::equals instead?
Don't we want to compare all RoleMember fields?

Reply:
I've also considered that, but the only fields that can be updated by the end user (like through the UI) are only these two fields. I don't want to check other things that can't be changed or are not allowed to be changed.